src/wallet/rpcwallet.cpp

```diff
@@ -5576,9 +5574,7 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
 
     unsigned int max_tx_size = saplingActive ? MAX_TX_SIZE_AFTER_SAPLING : MAX_TX_SIZE_BEFORE_SAPLING;
     size_t estimatedTxSize = 200;  // tx overhead + wiggle room
```

Comment:
```suggestion
    size_t estimatedTxSize = MIN_TX_COST;  // tx overhead + wiggle room
```

Reply:
No. We still don't support UAs in `z_mergetoaddress`, and the size of the Sprout or Sapling shielded output is accounted for right after this.

Reply:
No. We still don't support UAs in `z_mergetoaddress`, and the size of the Sprout or Sapling shielded output is accounted for right after this.